pom.xml

```diff
@@ -27,6 +27,7 @@
         <module>eclair-core</module>
         <module>eclair-node</module>
         <module>eclair-node-gui</module>
+        <module>sqlite2psql</module>
```

Comment:
I don't know if we want to have this tool in the eclair repository...

Reply:
We definitely need a SQLite -> PostgreSQL migration tool, otherwise there's no way to switch to Postgres for existing nodes.

I don't really like to have it as a separate project either. I'm open to suggestions how to package  it in a better way.